forkable/options.go

```diff
@@ -40,6 +43,17 @@ func WithExclusiveLIB(irreversibleBlock bstream.BlockRef) Option {
 	}
 }
 
+func WithIrreversibilityChecker(blockIDServer pbblockmeta.BlockIDServer, delayBetweenChecks time.Duration, maxNormalLIBDistance uint64) Option {
```

Comment:
Remove usage of `maxNormaLIBDistance` and use `GetNormaLIBDistance` global value.